x-pack/legacy/plugins/siem/public/pages/detection_engine/rules/edit/index.tsx

```diff
@@ -205,10 +205,10 @@ const EditRulePageComponent: FC = () => {
             : myAboutRuleForm.data) as AboutStepRule,
           (activeFormId === RuleStep.scheduleRule
             ? activeForm.data
-            : myScheduleRuleForm.data) as ScheduleStepRule,
-          ruleId
-        )
-      );
+            : myScheduleRuleForm.data) as ScheduleStepRule
+        ),
+        ...(ruleId ? { id: ruleId } : {}),
```

Comment:
rules have both an `id` and a `rule_id`, is this `ruleId` meant to represent the id or the `rule_id`?

Reply:
This is coming in from a route parameter on the edit page, so it should be `id` 👍